src/components/ScenarioParameters/components/ScenarioParametersInputs/GenericTable.js

```diff
@@ -678,11 +678,11 @@ export const GenericTable = ({
       // To trigger isDirty state when an already saved table was reverted and avoid it in other cases, we need to
       // updateParameterValue setter and updateOnFirstEdition function that triggers the start of edition; on the other
       // hand, updateParameterValueWithReset setter rollbacks modified values without triggering isDirty
-      _getDataFromTwingraphDataset(parameter.id ? updateParameterValue : updateParameterValueWithReset);
-      if (parameter.id) updateOnFirstEdition();
+      _getDataFromTwingraphDataset(parameter.content ? updateParameterValue : updateParameterValueWithReset);
+      if (parameter.content) updateOnFirstEdition();
     },
-    // eslint-disable-next-line
-    [parameter.id, updateParameterValue, updateParameterValueWithReset, updateOnFirstEdition]
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
yes, because the callback uses the internal function `_getDataFromTwingraphDataset` which will rerender the hook every time something changes. In order to avoid it, it's better to keep it out of dependencies array and we need to disable eslint warning for it because it doesn't allow a non exhaustive depth.